examples/bert.rs

```diff
@@ -42,34 +43,21 @@ impl Config {
     }
 }
 
+#[derive(Parser)]
+struct Args {
+    /// Prompt to run
+    #[arg(short, long, default_value_t = String::from("Stocks rallied and the British pound gained"))]
+    prompt: String,
```

Comment:
please let me know if this change is okay.

previously prompt was unnamed argument. With this change it is explicitly named argument with `-p` or `--prompt`

Reply:
It's ok !